src/libs/ascent/runtimes/ascent_vtkh_data_adapter.cpp

```diff
@@ -1891,68 +1893,97 @@ VTKHDataAdapter::AddField(const std::string &field_name,
         //   Array: valueType=x storageType=N4vtkm4cont15StorageTagBasicE 27 values occupying 216 bytes [0 1 2 ... 24 25 26]
         //   TypeList: N4vtkm4ListIJfdEEE
         // ***********************************************************************
+        // ***********************************************************************
+        // NOTE: int32 should work as of sept 10 2024 
+        // ***********************************************************************
```

Comment:
VTK-m 2.2, or later?

Reply:
It should have always worked, and it was working when I uncommented it, but the errors previously encountered make me too nervous to commit to changing it from something that works to something that might work sometimes. So I just left a note and if the else catch-all stops working for int32s we can try changing it.

Reply:
Got it, thanks for clarifying.